### PR TITLE
[SDK-430] Address export test findings

### PR DESCRIPTION
**Pull request by kopreschko**
- Adapted `experimental` decorator to support static methods
- Removed unnecessary argument `task_id` for methods in ExportTask

### CODE REVIEW

**Review by vbrodsky — APPROVED** [review]
with some optional suggestions
also a question: why do we bind the `experimental` wrapper to DbObject?

**Comment by kopreschko** [comment]
> with some optional suggestions also a question: why do we bind the `experimental` wrapper to DbObject?

not sure about that, that's where it was, but it doesn't seem to be the right place for it.

File: labelbox/orm/db_object.py [hunk 1]
```diff
@@ -270,14 +270,37 @@ def delete(self):
...
-            raise Exception(
-                f"This function {fn.__name__} relies on a experimental feature in the api. This means that the interface could change."
-                " Set `enable_experimental=True` in the client to enable use of experimental functions."
-            )
-        return fn(self, *args, **kwargs)
+    def wrapper(*args, **kwargs):
+        client = None
+        wrapped_fn = None
+        if is_static:
+            # assumes that the first argument is the client, needs modification if that changes
```
[the remark this answers is not in the document]
Comment (kopreschko) [reply labelbox/orm/db_object.py:284]: For now, it works for the current implementation (YAGNI). The @staticmethod that needs to be supported is the `export` method in DataRow. This is the only entity that uses export/export_v2 as static methods.

File: labelbox/orm/db_object.py [hunk 2]
```diff
@@ -270,14 +270,37 @@ def delete(self):
...
+            if len(args) >= 1:
+                client = args[0]
+            elif "client" in kwargs:
+                client = kwargs["client"]
```
[the remark this answers is not in the document]
Comment (kopreschko) [reply labelbox/orm/db_object.py:288]: Same answer as above.

**Pull request merged**